models/topics.model.js

```diff
@@ -0,0 +1,15 @@
+const format = require ('pg-format')
+const db = require ('../db/connection.js')
+
+
+const selectAllTopics = () => {
+let sqlString = `SELECT * from topics`
+return db.query(sqlString).then(({rows}) => {
+    if (rows.length === 0){
+        return Promise.reject({status: 404, msg: 'path not found'})
```

Comment:
not really necessary as either relation topics exists and is empty or psql error will be thrown and caught in your psql error handler in app.js.